bin/qless-py-worker

```diff
@@ -1,10 +1,11 @@
 #! /usr/bin/env python
 
-import psutil
+import multiprocessing as mp
 import argparse
 
 # First off, read the arguments
-parser = argparse.ArgumentParser(description='Run qless workers.')
+parser = argparse.ArgumentParser(description='Run qless workers.',
```

Comment:
Yes sorry, I didn't think github would mix my branches, so i didn't check what went into this pull request. only use 0e6e498 and ignore 13deeea